package.json

```diff
@@ -54,33 +86,41 @@
     "package-lock.json": "npm install --prefer-offline --no-audit"
   },
   "scripts": {
+    "check-policies": "TESTING=1 binzx/otomi check-policies",
+    "compile": "tsc && chmod +x ./dist/otomi.js",
+    "compile:watch": "npm run compile && tsc -w",
     "cz": "git-cz",
-    "check-policies": "TESTING=1 bin/otomi check-policies",
     "cz:retry": "git-cz --retry",
     "format": "prettier --check \"**/*.{json,md,yml,yaml}\"",
     "format:fix": "prettier --write \"**/*.{json,md,yml,yaml}\"",
     "husky:lint-staged": "lint-staged",
-    "husky:pre-push": "run-p husky:lint-staged lint spellcheck",
-    "lint": "TESTING=1 bin/otomi hf lint",
-    "lint:fix": "prettier --write tests/**/*.yaml --write '.values/env/**/*.yaml'",
-    "lint:all": "npx run-p spellcheck lint",
+    "husky:pre-push": "run-p husky:lint-staged lint:hf spellcheck",
+    "lint:hf": "TESTING=1 binzx/otomi hf lint",
+    "lint": "run-p spellcheck lint:hf lint:ts",
+    "lint:fix": "prettier --write tests/**/*.yaml --write '.values/env/**/*.yaml' && npm run lint:ts:fix",
+    "lint:ts": "eslint --ext ts .",
+    "lint:ts:fix": "eslint --fix --ext ts .",
+    "prepare": "husky install",
     "release": "standard-version",
     "release:github": "github-release-from-changelog",
+    "score-templates": "TESTING=1 binzx/otomi score-templates",
     "spellcheck": "cspell '/docs/**/**.md' '/values-schema.yaml' '/*.md' '/.values/README.md'",
-    "score-templates": "TESTING=1 bin/otomi score-templates",
     "test": "run-s spellcheck test:bats lint validate-templates check-policies",
-    "test:bats": "TESTING=1 bin/otomi x bats -T bin/tests",
-    "test:ci": "TESTING=1 bin/otomi x bash -c 'bin/ci-tests.sh'",
-    "test:opa": "bin/otomi x opa test policies -v",
+    "test:bats": "TESTING=1 binzx/otomi x bats -T bin/tests",
+    "test:ci": "TESTING=1 binzx/otomi x bash -c 'bin/ci-tests.sh'",
+    "test:opa": "binzx/otomi x opa test policies -v",
```

Comment:
do we need to add `TESTING=1` here?

Reply:
https://github.com/redkubes/otomi-core/blob/master/package.json#L73

Old CLI reasons